controllers/git_tekton_resources_renovater_test.go

```diff
@@ -127,5 +148,24 @@ var _ = Describe("Git tekton resources renovater", func() {
 			deleteComponent(componentNamespacedName3)
 			deleteComponent(componentNamespacedName4)
 		})
+
+		// test has to be the last, because it isn't waiting for reconcile action,
+		// so test which would ran after it would create secret again
+		It("It should trigger job, because pac secret is missing", func() {
```

Comment:
What does this test check? I see no asserts in it.

Reply:
asserting event now